functions/src/modules/SQR.functions.ts

```diff
@@ -278,6 +278,21 @@ export const processSubmission = functions.database
   if (allSubmissionsSnapshot.exists()) {
     const allSubmissions = allSubmissionsSnapshot.val();
 
+    let currentSet = (await db
+        .ref(`/files/${list}`)
+        .orderByChild("soundQualityReporting/assignee/emailAddress")
+        .equalTo(submission.author.emailAddress).once("value")
+    ).val();
+
+    currentSet = Object.entries(currentSet).map(
```

Comment:
I'm not completely understand this construct. Can't we use `forEach` to iterate the objects and alter them as needed instead of mapping into another array?

Reply:
All of them are altered because they need fileName as well as other properties.